### ISSUE TITLE
WCAG 2.1 (2023)/ WCAG 2.1 解説書のリリース準備（2023-11）

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#1859]
- 前の解説書/達成方法集のリリース：#1800

----

- [x] #1856 に対応する
- [x] リリース機能でリリースする
- [x] ステージング環境にアップロードする
- [x] WAICレビューのメールを主査が送信する
- [x] レビュー期間中のコメントに対する必要なアクションがないことを確認する
- [x] 本番環境で公開する
- [ ] 必要であればWAICのニュースとして告知する

リリースタグ：
 https://github.com/waic/wcag21/releases/tag/wcag21-202311

前回リリースからの差分：
https://github.com/waic/wcag21/compare/wcag21-202307...wcag21-202311

**Comment by bakera** [comment]
解説書はいったん置いておいて、本体だけ先に公開する方向で

**Comment by bakera** [comment]
リリースを作成
https://github.com/waic/wcag21/releases/tag/wcag21-202311

**Comment by bakera** [comment]
ステージング環境に反映しました。

**Comment by momdo** [comment]
12/5（火）締め切り目途と送信済み

**Comment by momdo** [comment]
- #1876 の対応を確認する

**Comment by momdo** [comment]
https://github.com/waic/wcag21/issues/1876 については、
 
- #1880 
- #1879 
- #1877

を作成済み。

- #1878 

については、どのように対応するのかを決める。

**Comment by bakera** [comment]
https://github.com/waic/wcag21/issues/1876 関連のPRはマージしました。

**Comment by bakera** [comment]
https://github.com/waic/wcag21/pull/1877#issue-2008902931
>＃ そういえば解説書4.1.1をそもそも更新していなかったですね…

これ注記をコピペするだけですかね?
他にも修正点あったりしますでしょうか……?

**Comment by bakera** [comment]
https://github.com/waic/wcag21/issues/1878
対応方針を決める必要があるがどうするか。いったんSlackで意見募集してみますか。

**Comment by momdo** [comment]
それぞれPRを作りました

- #1882
- #1881

**Comment by bakera** [comment]
ありがとうございます。マージしました。

**Comment by bakera** [comment]
フィードバック対応ぶんのリリース作成
https://github.com/waic/wcag21/releases/tag/wcag21-202312

**Comment by bakera** [comment]
本番反映しました。
https://waic.jp/translations/WCAG21/

解説書4.1.1の注記も反映されています。
https://waic.jp/translations/WCAG21/Understanding/parsing.html

**Comment by momdo** [comment]
https://waic.jp/news/20231208/　🎉
